### DISCUSSION TITLE
Suggested tags (Tumblr feature?) floating box: any way to get rid of it?

### DISCUSSION THREAD

**Discussion by Eregyrn** [AprilSylph/XKit-Rewritten#763] [Suggestion Box]
I only just installed XKit Rewritten.  In it, it appears that Quick Tags only works in the new post editor.  I've avoided using that until now, but I depend on Quick Tags, so I'm starting to use it.  Therefore, I'm not sure if what I'm about to ask is a Tumblr feature, or something I've inadvertently toggled on in XKR.  (I tried searching in here, but I can't find another answer about this, unless I'm using the wrong search terms.)

When I am reblogging in the new post editor, I noticed that this pop-up window with suggested tags comes up, and it's REALLY obtrusive.  I remember a suggested-tags pop-up even using the old editor, but it wasn't so persistent and it didn't position itself over some of the other existing tags.  I'm having a hard time figuring out how to make it go away or get out of the way so I can reach the tags beneath it.

If this is a Tumblr feature, is it possible for XKR to give us a way to toggle it off?  Does that exist already in XKR?  (If the ability to turn it off exists in Tumblr, I haven't found it yet.)  Or is this something I should just be contacting Tumblr support to ask about?

**Comment by marcustyphoon** [comment]
Pretty sure this is indeed part of Tumblr itself (if you wanted, you could confirm by temporarily disabling XKit and seeing if it still happens). When you say it covers previous tags, you mean because it's positioned above the cursor position vertically if it's tall enough, and so if you have more than one row of tags, the previous rows get covered? I agree, that is rather annoying.

Definitely write in to Tumblr support with your experiences; maybe there's a way to make it less intrusive! As far as XKit Rewritten goes, indeed I don't think we have a function to hide it, but it looks like it would be a pretty easy tweak to add, so this is totally a valid feature request.
